src/numberDisplayFormat.ts

```diff
@@ -184,8 +194,10 @@ function getMillifiedFormat(value: string | BigNumber): string {
 
 function getCommifiedFormat(
   actualValue: BigNumber | string,
-  decimalPlace = 2
+  decimalPlace = 2,
```

Comment:
change to `decimalPlace: number | 'exact' = 2` ?

Reply:
if (decimalPlace === 'exact') return `utils.commify(getStringInput(actualValue))`